content/en/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+In applications of robotics and automation, a _control loop_ is
+a non-terminating loop that regulates the state of the system.
+
```

Comment:
hello @sftim .
This is a good start to the Controller overview. I think this page is fairly important. Here are a few ideas/questions I have about `What is a Controller?`:
- How does the first sentence tie in to the k8s controller concept (the second sentence)?
- What is a built-in controller? How does a built-in controller differ from a custom controller?
-  Are the controllers executors as well as state maintainers?
- What happens if a controller fails? Is the controller restarted?
- The Controller pattern mentions `well-implemented`, but does not say how to implement one
- Add a `What's next` section to tie in additional controller resources, such as `kubebuilder`?
- Clients of controllers?